rest-server/src/components/project/__tests__/project_joiValidation.test.ts

```diff
@@ -36,9 +43,16 @@ beforeAll(async () => {
   await syncProjectTables();
   await request(app)
     .post(signupUrl)
-    .send({ email: 'test@gmail.com', password: 'test' });
+    .send({ email: 'newuser@gmail.com', password: 'test' });
 });
 
+describe('GET/api/projects joi validation tests', () => {
+  // test('checks request params to see if it is a number', async () => {
+  //   const { statusCode } = await request(app)
+  //     .get(`${projectUrl}/1`)
+  //   expect(statusCode).toBe(200);
+  // })
+})
 describe('POST/api/projects joi validation tests', () => {
   test('checks title for any unwanted characters', async () => {
     testProject.title = 'test!@#$%';
```

Comment:
instead of  manually resetting the [property] in the test obj after every test, you can use the beforeEach func provided by jest and set testProject equal to the original testProj
for example
beforeEach(() => testProject = { ...originalObject });

Reply:
oh you right